airgun/views/rhai.py

```diff
@@ -10,3 +12,28 @@ class AllRulesView(BaseLoggedInView):
     @property
     def is_displayed(self):
         return self.title.is_displayed and self.search.is_displayed
+
+
+class InventoryAllHosts(BaseLoggedInView):
+    search = TextInput(locator=".//input[@placeholder='Find a system']")
+    actions = Dropdown("Actions")
+    systems_count = Text(".//h3[@class='system-count']")
+    table = SatTable(".//table",
+                     column_widgets={"System Name": Text(".//a")})
+    export_csv = Button("Export CSV")
+
+    @property
+    def is_displayed(self):
+        return self.menu.currently_selected == ["Insights", "Inventory"]
```

Comment:
it is not a criteria for view displayed state as item selected in menu doesn't guarantee that something was actually opened. Please use our approach with a title, breadcrumb if you have it or any unique element that can be present on that specific page